test/python/circuit/test_controlled_gate.py

```diff
@@ -443,15 +659,14 @@ def test_open_controlled_unitary_matrix(self, num_ctrl_qubits):
                         (output_ctrl == input_ctrl) and
                         (output_target == cond_output))
                 else:
-                    self.assertTrue((
-                        (output_ctrl == input_ctrl) and
-                        (output_target != cond_output)) or
-                                    output_ctrl != input_ctrl)
+                    self.assertTrue(
+                        ((output_ctrl == input_ctrl) and (output_target != cond_output)) or
+                        output_ctrl != input_ctrl)
```

Comment:
Having different assert under a condition tells me that these should be two different tests. What do you think? Would it be possible to split these situations ?

Reply:
This test was already in the master branch (my editor probably changed the format without me noticing), I think these tests are from @ewinston. Probably the entire condition in the assert must be fulfilled, or can they be split?

Reply:
I put this into a single test because it is a validation of a truth table; the clause just separates truth and false conditions.